lib/paged-sync.js

```diff
@@ -56,7 +56,8 @@ export default function pagedSync (http, query, resolveLinks) {
     .then((response) => {
       // clones response.items used in includes because we don't want these to be mutated
       if (resolveLinks) {
-        mixinLinkGetters(response.items, mapIncludeItems(cloneDeep(response.items)))
+        response.includes = mapIncludeItems(cloneDeep(response.items))
+        response.items = resolveResponse(response, {resolveLinks, removeUnresolved})
```

Comment:
No need for this cloneDeep since resolve-response is already doing it:
https://github.com/contentful/contentful-resolve-response/blob/master/index.js#L92